.travis.yml

```diff
@@ -1,93 +1,44 @@
 ---
-language: generic
-
-addons:
-  apt:
-    packages:
-    - gettext
-    - libcurl4-openssl-dev
-    - libicu-dev
-    - libssl-dev
-    - libunwind8
-    - zlib1g
+language: csharp
+solution: mono/MaxMind.Db.Mono.sln
 
 env:
   global:
-    - MAXMIND_BENCHMARK_DB=MaxMind.Db.Benchmark/GeoLite2-City.mmdb
-    - MAXMIND_TEST_BASE_DIR=MaxMind.Db.Test
+    - MAXMIND_BENCHMARK_DB=$TRAVIS_BUILD_DIR/MaxMind.Db.Benchmark/GeoLite2-City.mmdb
+    - MAXMIND_TEST_BASE_DIR=$TRAVIS_BUILD_DIR/MaxMind.Db.Test
```

Comment:
This was required by mono jobs as the build.sh implicitly does `cd $TRAVIS_BUILD_DIR/dev-bin/../mono`, which makes our helper tries to locate the unit test directory inside mono. With donet CLI job, build.sh changes pwd to `$TRAVIS_BUILD_DIR/dev-bin/..`, therefore the test runs from those jobs pass without this change.